admin-manual/installation/installation.rst

```diff
@@ -861,6 +861,63 @@ Depending on your browser settings, you may need to clear your browser cache to
 make the dashboard pages load properly.  For example in Firefox or Chrome you
 should be able to clear the cache with control-shift-R or command-shift-F5.
 
+* Update Transfer Index
+
+Archivematica 1.6.0 includes two new features that make use of the Transfer 
+Backlog - the Appraisal Tab and the Backlog Tab.  At the present time, neither
+of these features is recommended to be used with Transfers that were put into
+backlog with the 1.5.x release.  They are both safe to use with new Transfers
+created with Archivematica 1.6.0.
+
+If you are interested in experimenting with the use of these new features, 
+with a backlog created in an earlier version of Archivematica, these 
+instructions show how to update your Transfer Backlog Index so it can be 
+used with the Appraisal Tab and the Backlog tab.
+
+These are experimental instructions. Do not use them on a production system
+unless you have a back you can restore from. 
+
```

Comment:
Lines 866 to 878 can now be removed, and the equivalent from the deb instructions above.